src/save_index.cpp

```diff
@@ -230,6 +231,22 @@ class filename_filter
 	std::string filter_;
 };
 
+/** Ignore certain files - in particular, the auto-generated Steam cloud record. */
+class filename_ignore
+{
+public:
+	bool operator()(const std::string& filename) const
+	{
+		return std::find(files_to_ignore_.begin(), files_to_ignore_.end(), filename) != files_to_ignore_.end();
+	}
+
+private:
+	/* 'The file steam_autocloud.vdf will be created in each location specified by your Steamworks cloud paths.
+		This file is used by Steam, and can be ignored by your game.'
```

Comment:
Maybe paraphrase this two lines instead of copying them verbatim? I don't think it matters, but means no-one will wonder about the copyright.

Reply:
Why would copyright be a concern? I was simply trying to show that the Steam documentation makes it clear that we (the game) can ignore this auto-generated file and I assume that most people don't bother clicking through links (plus on-line documentation can change too). I can change this, but I don't see a compelling reason to do so given what I was trying to do here.

Reply:
Copyright is a concern because we distribute Wesnoth; I don't think such a small quote matters but copyright's something where it's easier to avoid the question than to work out what the answer to the question is.

How about "Metadata that's automatically created by the Steam cloud-save framework, not a game that can be loaded"?

Reply:
My question was more along the lines of how copyright is a concern since I was only quoting public documentation - doesn't the US have something called 'fair use' as well? Plus this is only commentary, not actual code.

But if it seems this is such a cause for concern I'll rephrase this shortly.